.env.example

```diff
@@ -43,3 +43,9 @@ LEGACY_AUTH_TOKEN=longvalueofnumbersandletters.inbothcases.splitin3by2periods
 CHARGING_MODULE_TOKEN_URL=https://myinstance.amazoncognito.com
 CHARGING_MODULE_TOKEN_USERNAME=valuefullofnumbersandlettersinlowercase
 CHARGING_MODULE_TOKEN_PASSWORD=longvaluefullofnumbersandlettersinlowercase
+
+# AWS S3 config
+AWS_REGION=eu-west-1
+AWS_ACCESS_KEY_ID=uploadaccesskey
+AWS_SECRET_ACCESS_KEY=uploadsecretkey
+AWS_MAINTENANCE_BUCKET=upload-bucket-gov-uk
```

Comment:
Just to avoid the scary 'no entry' sign in GitHub we try and ensure all files end with a new line.

![Screenshot 2023-05-16 at 12 48 37](https://github.com/DEFRA/water-abstraction-system/assets/1789650/3462a0fa-1d9d-48a2-b274-8e200294bce6)

I think there is even a setting in VSCode that will ensure this is there when you save a file.

```suggestion
AWS_MAINTENANCE_BUCKET=upload-bucket-gov-uk

```